cli/flox-rust-sdk/src/providers/services.rs

```diff
@@ -89,46 +90,30 @@ pub fn write_process_compose_config(
     Ok(())
 }
 
-/// A container for the path to the `process-compose` config file
-///
-/// This is necessary in the situation where the config file is written to a temporary location
-/// and we need to keep the destructor from deleting the temp directory before we're done with it.
-#[derive(Debug)]
-pub struct ConfigPath {
-    pub path: PathBuf,
-    _tempdir: Option<tempfile::TempDir>,
-}
-
 /// Determines the location to write the `process-compose` config file
-pub fn process_compose_config_write_location() -> Result<ConfigPath, ServiceError> {
+pub fn process_compose_config_write_location(
```

Comment:
```suggestion
pub fn service_config_write_location(
```
Nitpick: we've kept `process-compose` out of interfaces in most other places, the exception being `write_process_compose_config` because it does a `process-compose`-specific thing (rendering a `ManifestServices` as a `process-compose` YAML config).

Reply:
Done, plus docstring.